packages/devtools-launchpad/webpack.config.devtools.js

```diff
@@ -20,7 +20,7 @@ module.exports = (webpackConfig, envConfig) => {
     webpackConfig.output.path = outputPath;
   }
 
-  webpackConfig.devtool = false;
+  // webpackConfig.devtool = false;
```

Comment:
I find it better to just remove code rather than comment it out.  The other hunks in this file seem unrelated.